### ISSUE TITLE
Browsers not always open the default page

### ISSUE THREAD

**Issue by nerdbeere** [james-proxy/james#78]
For some reasons the browsers not always open the defined default page.
It nevers works for me on chrome but always works on firefox it seems.

Needs further investigation.

**Comment by mitchhentges** [comment]
After ~6 hours of finding an `iMac`, and slowly upgrading it from Mac 10.4 to 10.11, I've finally found that: I can't reproduce this issue.

I've installed both Firefox and Chrome. Upon launching Chrome from James, it starts loading `http://uxebu.com`, and the requests appear in James:
![chromeok](https://cloud.githubusercontent.com/assets/7784737/12757976/ae3ae848-c9da-11e5-95b3-1f669b743012.gif)

**Comment by nerdbeere** [comment]
Chrome is my main browser so there's always an instance open that doesn't use james.

It seems that when you open a second instance of chrome through james it won't open the default page.

**Comment by mitchhentges** [comment]
Ah! I'm able to reproduce it (mac-only), and here's my understanding:

There's two types of instances of Chrome:
1. There's a primary window of Chrome, which is launched when the user starts Chrome as normal
2. There's an "alternate" window of Chrome, which is launched by James

If you have a primary window open, and you open Chrome through James (the first time), it will work. However, upon launching _more than than_ of the second type of instance, the new windows won't go to the requested URL.

Even if James is restarted, if the second instance of Chrome isn't completely killed (`Command-Q`), then successive Chrome starts won't load the intended URL.

I'm not 100% sure why this doesn't occur on Linux. My Linux machine's behaviour is, upon each click of the Chrome icon in James, to open a separate window of my primary instance, then to add tabs (Mac _always_ opens a new window, when clicking the icon in James).

Firefox always opens a new window, __in a new process__, so it is exempt from this bug.

TL;DR `browser-launcher2` has a bug

**Comment by mitchhentges** [comment]
Turns out that `google-chrome` is the source of the issue. I've reported a bug on the Chromium issue tracker: https://code.google.com/p/chromium/issues/detail?id=584839

**Comment by mitchhentges** [comment]
@julian can this be closed?

**Comment by nerdbeere** [comment]
Unfortunately this is something we can't fix.
